### DISCUSSION TITLE
Large Runner/Runner Pricing Feedback

### DISCUSSION THREAD

**Discussion by 24601** [community/community#41314] [Actions]
### Select Topic Area

Product Feedback

### Body

I really appreciate value-added services and am willing to pay for them, so I think GitHub has every right to price the convenience of their GitHub Actions minutes at a premium to bare compute. Not only is it convenient, but supported, with good images (although on this point, there are many of OSS lineage that are equally good). 

However, I must say, after comparing a trial of GH-hosted runners versus self-hosted runners on our own k8s fabric with ARC and finding up to 15x markups against bare compute (yes, I know, support costs, etc, even considering these...), I think for the part of the market that already has DevOps infra in house (and resources on the team internally to run it, which tends to be most folks using 64 vCPU and 256GB boxes I think), it's quite an enormous premium that, at least for me, is kind of absurd. 

When we can get r5a.12xlarge boxes for about $1/hr. GH runners that are comparable run $15/hr. A 15x multiplier is quite steep for organizations that are basically already paying for this infra (either on a cloud provider or in house) and teams who can manage it, and it comes with the added benefits of locality and avoiding possible bandwidth costs. 

I know this pricing was likely heavily studied and informed with lots of market analysis, so just sharing our one data point after our bake off. We aren't an enormous team, so we don't have tons of spare resources, so I imagine this cost:benefit equation is even more skewed for organizations who get deeper compute/memory discounts than we do and have deeper benches than we do.

**Comment by vitalykarasik** [comment]
>When we can get r5a.12xlarge boxes for about $1/hr. GH runners that are comparable run $15/hr
As far as I understand, on-demand pricing for r5a.12xlarge is about $2.7/hour, so if we compare it with GitHub runner with a similar number of CPUs, we'll see ~ 2 times difference.
Obviously doesn't make sense to compare with on-prem hardware.

From my experience - self-hosted runners make sense when we use a custom image with heavy non-standard stuff. In all other cases, I vote for managed runners. 

(of course - IMHO. I'm not working for GitHub).
